_includes/specials/taproot/en/03-p2wpkh-to-p2tr.md

```diff
@@ -0,0 +1,120 @@
+For wallets that already support receiving and spending v0 segwit P2WPKH
+outputs, upgrading to v1 segwit P2TR for single-sig should be easy.
+Here are the main steps:
+
+- **Use a new BIP32 key derivation path:** you don't need to change your
+  [BIP32][] Hierarchical Deterministic (HD) code and your users don't
+  need to change their seeds.[^electrum-segwit]  However, you are
+  strongly encouraged to use a new derivation path for your P2TR public
+  keys (such as defined by [BIP86][]); if you don't do this, there's a
+  [possible attack][bip340 alt signing] that can occur if you use the
+  same keys with both ECDSA and [schnorr signatures][topic schnorr
+  signatures].
+
+- **Tweak your public key by its hash:** although technically not
+  required for single-sig, especially when all your keys are derived
+  from a randomly-chosen BIP32 seed, BIP341 [recommends][bip341 cite22]
+  having your key commit to an unspendable scripthash tree.  This is as
+  simple as using an Elliptic Curve addition operation that sums your
+  public key with the curve point of that key's hash.  Advantages of
+  complying with this recommendation are that you'll be able to use the
+  same code if you later add scriptless [multisignature][topic
+  multisignature] support or if you add support for [`tr()`
+  descriptors][].
+
+- **Create your addresses and monitor for them:** use [bech32m][topic
+  bech32] to create your addresses.  Payments will be sent to the
+  scriptPubKey `OP_1 <tweaked_pubkey>`.  You can scan for transactions
+  paying the script using whatever method you use to scan for v0 segwit
+  addresses like P2WPKH.
+
+- **Creating a spending transaction:** all the non-witness fields for
+  taproot are the same as for P2WPKH.  The only difference for a
+  single-sig P2TR is that you'll only have one witness element (a
+  signature) instead of the two for P2WPKH (a key and a signature).
```

Comment:
It seems a bit premature to be talking about the witness here, when it only gets computed in the next two steps.